### PR TITLE
Renew react

### PR DESCRIPTION
**Pull request by AWare**
React UI for the renew flow (no endpoint in this branch) 
image

### CODE REVIEW

File: app/views/account/renew.scala.html [hunk 1]
```diff
@@ -56,36 +54,29 @@ <h3 class="mma-section__header">
...
+                    <h3 class="mma-section__header">
+                        Subscription end date
+                    </h3>
+                    @prettyDate(subscription.termEndDate)
+                </section>
+                <section class="mma-section">
+                    <h3 class="mma-section__header">Renew</h3>
+                    @helper.CSRF.formField
+                    <p>
+                        Here is some copy about renewing your subscription, which is a very fulfilling thing to do and should probably fill one to two lines of text telling the user what renewing their subscription entails.
```
Comment (johnduffell) [inline app/views/account/renew.scala.html]: I guess we'll fill this in later

File: app/views/fragments/javascriptFirstSteps.scala.html [hunk 2]
```diff
@@ -42,11 +42,12 @@
     ) {
     guardian.stripeCheckout = {
         key: '@backend.paymentService.stripeService.publicKey',
-        image: '@hashedPathFor("images/logos/gu.png")',
+        image: 'https://d24w1tjgih0o9s.cloudfront.net/gu.png',
```
Comment (johnduffell) [inline app/views/fragments/javascriptFirstSteps.scala.html]: is cloudfront the way we normally do this?  you probably know better than me!

File: assets/javascripts/modules/react/directDebit.jsx [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        super(props);
+    }
+
+    render() {
+        return <div>
+            <SortCodeComponent value={this.props.sortCode} onChange={this.props.handleSortCode} valid={this.props.validSortCode}/>
+            <AccountNumber onChange={this.props.handleAccountNumber} value={this.props.AccountNumber} valid={this.props.validAccountNumber}/>
+            <AccountHolder onChange={this.props.handleAccountHolder} value={this.props.AccountHolder} valid={this.props.validAccountHolder}/>
+            <Confirmation checked={this.props.directDebitConfirmed} onChange={this.props.handleDirectDebitConfirmation} valid={this.props.validDirectDebitConfirmed}/>
+            <dd className="mma-section__list--content"><p>This is some copy about Direct Debit and our legal
```
Comment (johnduffell) [inline assets/javascripts/modules/react/directDebit.jsx]: tbc I guess again

File: assets/javascripts/modules/react/directDebit.jsx [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+
+class AccountHolder extends React.Component {
+    constructor(props) {
+        super(props)
+
+    }
+
+    /*
+     * BACS requirement:
+     "The payer’s account name (maximum of 18 characters).
```
Comment (johnduffell) [inline assets/javascripts/modules/react/directDebit.jsx]: good comment! although do we actually limit the length here?

File: assets/javascripts/modules/react/directDebit.jsx [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            <dd className="mma-section__list--content">
+                <label className="option">
+                <span className="option__input">
+                    <input type="checkbox"
+                           onChange={this.props.onChange}
+                           checked={this.props.checked}
+                    />
+                </span>
+                    <span className="option__label">
+                    I confirm that I am the account holder and I am solely able to authorise debit from the account
```
Comment (johnduffell) [inline assets/javascripts/modules/react/directDebit.jsx]: wording waiting for approval too?

**Comment by johnduffell** [comment]
ok I am in no way qualified to comment on this one, not sure if anyone else wants to have a look! but from what I can see, 👍

File: assets/javascripts/modules/react/directDebit.jsx (hunk 5 above)
Comment (AWare) [inline assets/javascripts/modules/react/directDebit.jsx]: Wording from the checkout flow.

File: app/views/fragments/javascriptFirstSteps.scala.html (hunk 2 above)
Comment (AWare) [inline app/views/fragments/javascriptFirstSteps.scala.html]: Discussed the location with @paulbrown1982, we need to use the same icon for membership and card updates in profile.

File: assets/javascripts/modules/react/directDebit.jsx (hunk 5 above)
Comment (paulbrown1982) [inline assets/javascripts/modules/react/directDebit.jsx]: There might be other text which needs to be brought over too. The renewal flow will need to be signed off by the legal team.

**Comment by johnduffell** [comment]
PS I don't like that slidey switch for direct debit or credit card, I much prefer radio buttons, not sure what UX would say though ? maybe a task for post merge though

**Pull request merged**